packages/sync/src/utils/calculatePageLimit.ts

```diff
@@ -0,0 +1,49 @@
+interface DynamicLimiter {
```

Comment:
In Tamanu mobile we've removed the dynamic limiter because streaming replaces the need for it